mobile/src/components/common/text/text.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { selectFontColor, selectFontUsage } from './helpers';
+
+type Props = {
+  innerText: string;
+  style: {
+    fontColor: 'gray_400' | 'gray_200' | 'gray_100';
```

Comment:
After declaring enum mentioned in previous comment you could simplify type definition by using that enum